yacs/tests.py

```diff
@@ -24,6 +24,17 @@ def get_cfg():
     cfg.MODEL = CN()
     cfg.MODEL.TYPE = "a_foo_model"
 
+    # Some extra stuff to test CfgNode.__str__
```

Comment:
We can consider having a unit test for this in the future to avoid the overhead of checking the output.

Reply:
Now seems like the right time in the future :D. I added tests for `__str__` and `__repr__`.